src/bin/vip-cache-purge.js

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env node
+
+/**
+ * @flow
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import { appQuery, purgeCache } from 'lib/api/cache-purge';
+import command from 'lib/cli/command';
+import { trackEvent } from 'lib/tracker';
+import { readFromFile } from 'lib/read-file';
+import * as exit from 'lib/cli/exit';
+
+const examples = [
+	{
+		usage: 'vip @123.production cache purge <URL>',
```

Comment:
Should this be URLs, based on the design session and the list below in the command?

Reply:
I'm open to discussion, but I think I would prefer a single entrypoint to purging multiple URLs. The `--from-file` option makes the most sense to me. If I'm bringing a list of URLs to this command, it makes sense that I'd have them in a newline-separated list instead of a space-separated list.

So I'd argue for keeping this usage the way it is and only allowing one URL:

```
vip @123.production cache purge <URL>
```

For multiple URLs, users can use `--from-file` or just run the command multiple times:

```
vip @123.production cache purge <URL1>
vip @123.production cache purge <URL2>
```

Reply:
I think I'd also advocate for renamed the command to `purge-url`. I can envision adding other commands in the future; perhaps purging via a `BAN` directive (pattern) or potentially `purge-all` or purging other types other types of cache. So being specific here will help us avoid ambiguity in the future.

Reply:
@chriszarate I agree, I'm renaming it and removing the list of URLs using args.